sematic/ui/src/components/tests/test_RunStateChip.cy.tsx

```diff
@@ -1,5 +1,5 @@
 import RunStateChip from "../RunStateChip";
-import run from "../../../tests/fixtures/run.js";
+import run from "../../../tests/fixtures/run";
```

Comment:
Could you please do an absolute import instead?

Reply:
+1

Reply:
Thanks for mentioning that. Done.